examples/src/operation/swap/fixedOutputSwap.ts

```diff
@@ -30,42 +30,33 @@ export async function fixedOutputSwap({
    * Swap.getQuote method, which will return the best quote (highest rate)
    * after checking both v1 and v2
    */
-  const fixedOutputSwapQuote = Swap.v2.getQuote(
-    SwapType.FixedOutput,
+  const fixedOutputSwapQuote =  await  Swap.v2.getQuote({
```

Comment:
These examples use the swap router by default. Can you please an example that does not use the swap router for both fixedInputSwap and fixedOutputSwap.